guides/monomorphic-data-list.md

```diff
@@ -0,0 +1,32 @@
+# What
+
+Back in 2019, the [CLC has approved](https://gitlab.haskell.org/ghc/ghc/-/issues/20025#note_363426) the monomorphisation of `Data.List`.
```

Comment:
```suggestion
Back in 2019, the [CLC approved](https://gitlab.haskell.org/ghc/ghc/-/issues/20025#note_363426) the monomorphisation of `Data.List`.
```
(From what I understand about English grammar, pointing to an event in the past requires past simple, but feel free to disregard, of course.)